Kudu.Contracts/SiteExtensions/SiteExtensionInfo.cs

```diff
@@ -10,11 +9,12 @@ namespace Kudu.Contracts.SiteExtensions
     // This is equivalent to NuGet.IPackage
     public class SiteExtensionInfo : INamedObject
     {
-        public enum SiteExtensionType
+        public static class SiteExtensionType
```

Comment:
didn`t know this exist. the main reason is to update "Type" to return string value to caller.

let me give it a try to see what value we will return if we keep using enum with StringEnumConverter